doc/whats-new.rst

```diff
@@ -51,8 +54,8 @@ v2023.03.0 (March 22, 2023)
 ---------------------------
 
 This release brings many bug fixes, and some new features. The maximum pandas version is pinned to ``<2`` until we can support the new pandas datetime types.
-Thanks to our 19 contributors:
-Abel Aoun, Alex Goodman, Deepak Cherian, Illviljan, Jody Klymak, Joe Hamman, Justus Magin, Mary Gathoni, Mathias Hauser, Mattia Almansi, Mick, Oriol Abril-Pla, Patrick Hoefler, Paul Ockenfuß, Pierre, Shreyal Gupta, Spencer Clark, Tom Nicholas, Tom Vo
+Thanks to our 20 contributors:
```

Comment:
Don't change the content of the last version.
When the next version will be released (in ~1month) this paragraph will be added again for the next release.

Reply:
Oh, I thought that it was for this version. Sorry